IntroToCSharp/Pages/Activities/SettingBuilder.razor

```diff
@@ -0,0 +1,150 @@
+    </Text>
+    <Text>
+        In the following activity we are going to create a program that will help us create a
+        a city or town for our first game to take place. Through this program we will work on
+        working with the <Keyword Word="Console"/> to both prompt the user to enter in an input
+        and to store the input that the user input as a string or an integer.
+    </Text>
+</Section>
+    <Section Title="Learning Objectives">
+    <MudList Clickable="true">
+        <MudListItem Icon="@Icons.Filled.Circle">Practice using <Code>Console.WriteLine</Code> method.</MudListItem>
+        <MudListItem Icon="@Icons.Filled.Circle">Practice Working with <Code>strings</Code>.</MudListItem>
+        <MudListItem Icon="@Icons.Filled.Circle">Practice Working with <Code>integers</Code>.</MudListItem>
+        <MudListItem Icon="@Icons.Filled.Circle">Learn how to change the color of the terminal.</MudListItem>
+    </MudList>
+</Section>
+
+<Section Title="Where to start?">
+    <Text>
+        When we think of many of our favorite games what comes to mind? Perhaps you think of an astronaut
+        flying through an astreroid field in space like in No Man's Sky. Or maybe you think of a plumber jumping
+        on evil turtles to Store a princess like Mario. Maybe it is even something scarier where you are trapped
+        in a Kid's Pizza store with animatronics trying to come to life and get you!
+    </Text>
+    <Text>
+        There are so many different places that a game might take place but the first thing we should figure out
+        is what kind of "genre" the game is in. Genre is a term that generally means the particular "style" and
+        "world" of the game. By knowing our genre we know what places <i>could</i> be possible and what places 
+        wouldn't be possible. We wouldn't expect to see a space level in a fantasy game like Skyrim!
+    </Text>
+    <Text>
+        To help the user figure out what genre the game is in, let's give them a list of options that they can pick from.
+        You can use the following list as a base but feel free to add more if you want!
+    </Text>
+    <Note>
+        <b>Extra Challenge!</b><br>
+        Can you figure out how to add bullet points to your list as well? Hint: You should check out the ASCII lesson
+        for a possible value that might work!
+    </Note>
+    <MudList>
+        <MudListItem Avatar="@Icons.Filled.MenuBook">Fantasy</MudListItem>
+        <MudListItem Avatar="@Icons.Filled.MenuBook">Horror</MudListItem>
+        <MudListItem Avatar="@Icons.Filled.MenuBook">Adventure</MudListItem>
+        <MudListItem Avatar="@Icons.Filled.MenuBook">Sci-fi</MudListItem>
+        <MudListItem Avatar="@Icons.Filled.MenuBook">Post-Apocalyptic</MudListItem>
+        <MudListItem Avatar="@Icons.Filled.MenuBook">Mystery</MudListItem>
+    </MudList>
+</Section>
+    <TaskItem Name="Create Project"><Code>Create a project called "Setting Builder"</Code></TaskItem>
```

Comment:
No need to use <Code> blocks on task items. Also, consider using the <ConsoleProjectTask> which puts a button that allows them to see a cheat sheet for VS Code for setting up a project.

Reply:
Sorry, could you rephrase this? I don't understand what you are saying.

Reply:
You don't need the code blocks. Can be re-written as:
```
<TaskItem Name="Some Name">Task Goes here</TaskItem>
```

Also, there is a special component for doing "Create Project" tasks:
```
<ConsoleProjectTask Name="Create Project>Create a project called "Stting Builder"</ConsoleProjectTask>
```